code/modules/mapfluff/centcom/nuke_ops.dm

```diff
@@ -36,10 +36,27 @@
 
 /obj/machinery/door/puzzle/keycard/syndicate_chem
 	name = "Syndicate Chemical Manufacturing Plant"
-	desc = "Locked. Looks like you'll need a special access key to get in"
+	desc = "Locked. Looks like you'll need a special access key to get in."
 	puzzle_id = "syndicate_chem"
 
 /obj/machinery/door/puzzle/keycard/syndicate_fridge
 	name = "The Walk-In Fridge"
 	desc = "Locked. Lopez sure runs a tight galley."
 	puzzle_id = "syndicate_fridge"
+
+/// Terminals
+/obj/machinery/computer/terminal/infiltrator
+	desc = "A terminal running the latest build of SYNDIX. Whatever technician set this one up \
+	configured it to only run via command line, and without any GUI. How will you play games on it now?"
+	tguitheme = "syndicate"
+	upperinfo = "YOU ARE RUNNING SYNDIX ON REVISION #31410. REPORT ANY BUGS PROMPTLY." // References the (/tg/station) PR plastitanium smoothing was added!
+	content = list(
+		"<b>C:/Users/ADMIN/Desktop/'Chords, Read Me.txt'</b><br> \
+		<br> \
+		Chords, I'm sorry about our last night out. I know you didn't like what you saw, and I know I dissapointed you, \
+		greatly, when I acted the way I did - did what I did. I'm sorry, Chords, I can't hammer that home enough. \
+		I've spent days now trying to think of something to say, trying to think of some way to fix things, and I... \
+		I can't find any other than to sit here, in this shitty, dingy ass maintenance corridor, and just... type. \
+		Please. I'll remember to put on my oxygen mask AND run internals properly next operation. Talk to me again.<br> \
+		<br> \
+		> sudo rm -rf") // This one goes out to every flukie who's forgotten their internals, including myself. Godspeed you dumb, dumb dweebs.
```

Comment:
in before loser nerds make pr's to change either this or the windows file path name for inconsistencies

Reply:
Unintentional but incredibly funny, dork bait

Reply:
yeah let it be known they shall all be closed